setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup, find_packages
 
 here = os.path.abspath(os.path.dirname(__file__))
-README = open(os.path.join(here, 'README.rst')).read()
+README = open(os.path.join(here, 'README.md')).read()
```

Comment:
Any reason for going tom .rst to md. I thought rst is common with python projects.

Reply:
encoded was md, so I made this follow suite.